hack/boilerplate/boilerplate.py.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright YEAR Caicloud Authors
```

Comment:
we already have this under root directory.

Reply:
i move it into `hack` package. and add python and sh boilderplate